test_jit fails in test_freeze_conv_relu_fusion with
Mismatched elements: 7 / 30 (23.3%)
Greatest absolute difference: 3.053247928619385e-05 at index (1, 1, 0, 0, 0) (up to 1e-05 allowed)
Greatest relative difference: 0.0004548609140329063 at index (3, 1, 0, 0, 0) (up to 1.3e-06 allowed)

Increase the tolerance to allow this to pass.

Author: Alexander Grund (TU Dresden)

diff --git a/test/jit/test_freezing.py b/test/jit/test_freezing.py
index c8c1441adbf..e0feffd6bb5 100644
--- a/test/jit/test_freezing.py
+++ b/test/jit/test_freezing.py
@@ -2733,7 +2733,7 @@ class TestFrozenOptimizations(JitTestCase):
                     else:
                         FileCheck().check("aten::cudnn_convolution_relu").run(frozen_mod.graph)
 
-                self.assertEqual(mod_eager(inp), frozen_mod(inp))
+                self.assertEqual(mod_eager(inp), frozen_mod(inp), atol=1e-4, rtol=4e-3)
 
     @unittest.skipIf(not (TEST_CUDNN or TEST_WITH_ROCM), "requires CUDNN")
     def test_freeze_conv_relu_fusion_not_forward(self):
